.github/workflows/build-test-distribute.yml

```diff
@@ -268,12 +268,21 @@ jobs:
   update-dev-documentation:
     if: ${{ github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'full-ci') || contains(github.event.pull_request.labels.*.name, 'update-doc') }}
     needs:
-      - versioning-and-release-url
+      - ubuntu-arm64-build-test
     uses: ./.github/workflows/update-docs.yml
     with:
       output_folder: MeshLib/dev
     secrets: inherit
 
+  update-documentation:
+    if: ${{ github.event_name == 'release' && github.event.action == 'published' }}
+    needs:
+      - ubuntu-arm64-build-test
+    uses: ./.github/workflows/update-docs.yml
+    with:
+      output_folder: MeshLib
+    secrets: inherit
```

Comment:
this will never be called, because this workflow is not triggered on release

Reply:
reworked